website/versioned_docs/version-7.0.0/features/lambda_dead_letter_queue.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
It looks like when you created the website versioned docs, you used `7.0.0` as the version. By convention, we use `v7.0.0`. It doesn't actually matter - the docs would publish either way. However, for consistency with other versions we might want to update to use `v7.0.0`:

https://nasa.github.io/cumulus/versions

Reply:
[`f0150a0` (#2112)](https://github.com/nasa/cumulus/pull/2112/commits/f0150a05f79556cbe6b08f91754f3c8b45825f08)